src/imp/tracer_imp.js

```diff
@@ -1199,8 +1200,8 @@ export default class Tracer extends opentracing.Tracer {
                     }
 
                     if (res.commandsList && res.commandsList.length > 0) {
-                        if (res.commandsList[0].devMode) {
-                            this.options().disable_meta_event_reporting = false;
+                        if (res.commandsList[0].devMode && this.options().disable_meta_event_reporting !== true) {
```

Comment:
(we can just ensure that the devmode command is always the first command, I suppose)

Reply:
The object we're getting back looks like this -
```
[ {
    devMode: true,
    disabled: false,
}]
```
Any additional commands we add would also be in that object, which there's only one of. The position of the `devMode` field doesn't matter, as we're getting the only object, and checking for a field on an object will result in the following -
1. The key doesn't exist, which returns 'undefined', which is falsey in an `if` statement.
2. The key does exist, and its value is false, which returns 'false'.
3. The key does exist, and its value is true, which returns 'true'.

Reply:
I was assuming we'd use one command per command, otherwise we shouldn't have a list of commands in the response.

```
[ {
  devMode: true
}, {
  disable: true
}, {
  logThis: "you should update your tracer, dude"
} ]
```

Reply:
Right now we're only using one command per message.